applications/serial_lte_modem/src/main.c

```diff
@@ -411,10 +401,17 @@ int main(void)
 	if (slm_settings_init() != 0) {
 		LOG_WRN("Failed to init slm settings");
 	}
+
+	err = nrf_modem_lib_init(NORMAL_MODE);
+	if (err && (fota_stage == FOTA_STAGE_INIT)) {
```

Comment:
Maybe `if (err < 0)`, making the code more readable.
There should be impossible for `nrf_modem_lib_init()` to return DFU result while FOTA does not happen.